src/models/permissions.js

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+module.exports = (sequelize, DataTypes) => {
+  const Permission = sequelize.define('Permissions', {
+    name: { type: DataTypes.STRING, primaryKey: true }
+  }, {
+    timestamps: false,
+    classMethods: {
+      associate: db => {
+        Permission.belongsToMany(db.Clients, { through: 'ClientPermissions' });
```

Comment:
Yes, a permission can be applied to multiple clients and a client can have multiple permissions. In the future we may want to be able to query all clients with a specific permission.